armbalancer.go

```diff
@@ -82,8 +85,17 @@ type recyclableTransport struct {
 func newRecyclableTransport(id int, parent *http.Transport, host string, recycleThreshold, minReqsBeforeRecycle int64) *recyclableTransport {
 	tx := parent.Clone()
 	tx.MaxConnsPerHost = 1
+	hostUrl, err := url.Parse("https://" + host)
```

Comment:
Please use `net.SplitHostPort`  to avoid prepending `"https://"`

Reply:
fixed in new commit